package.json

```diff
@@ -19,10 +19,11 @@
     "test": "builder run npm:test"
   },
   "dependencies": {
-    "builder": "^2.0.1",
-    "builder-react-component": "0.0.4"
+    "builder": "2.1.0",
```

Comment:
The only reason I ask is that I've missed this on projects before and it's burned me. As long as it was intentional, that's fine...I just hate having to do version bumping across multiple projects or on one project if I don't need to. Since you're following semver, it should be safe to use `^`.